upgrade/v1beta1/upgrade.go

```diff
@@ -0,0 +1,186 @@
+package v1beta1
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jinzhu/copier"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	restclient "k8s.io/client-go/rest"
+
+	longhorn_v1beta1 "github.com/longhorn/longhorn-manager/upgrade/v1beta1/k8s/pkg/apis/longhorn/v1beta1"
+	lhclientset_v1beta1 "github.com/longhorn/longhorn-manager/upgrade/v1beta1/k8s/pkg/client/clientset/versioned"
+	"github.com/longhorn/longhorn-manager/upgrade/v1beta1/types"
+
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
+	lhclientset "github.com/longhorn/longhorn-manager/k8s/pkg/client/clientset/versioned"
+)
+
+const (
+	upgradeLogPrefix = "upgrade from v1beta1 to v1beta2:"
+)
+
+func UpgradeFromV1beta1ToV1beta2(config *restclient.Config, namespace string, lhClient *lhclientset.Clientset) error {
+	lhClientV1beta1, err := lhclientset_v1beta1.NewForConfig(config)
```

Comment:
Consider add at the function `UpgradeFromV1beta1ToV1beta2` beginning, so, all the below `return err` will have the same log prefix. [1]
```go
        defer func() {
		err = errors.Wrapf(err, upgradeLogPrefix+" failed")
	}()
```